spring-cloud-app-broker-deployer-cloudfoundry/src/test/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryAppDeployerUpdateApplicationTest.java

```diff
@@ -160,6 +172,13 @@ void setUp() {
 		given(operationsApplications.get(any()))
 			.willReturn(Mono.just(createApplicationDetail()));
 
+		given(applicationsV2.summary(any(SummaryApplicationRequest.class)))
+			.willReturn(Mono.just(SummaryApplicationResponse.builder()
+				.id(APP_ID)
+				.name(APP_NAME)
+				.services(Collections.emptyList())
+				.build()));
```

Comment:
This mock was for getting application summaries for the new `AppDeployer.get()` implementation.  Probably don't need this either.

Reply:
removing this one too...